index.js

```diff
@@ -443,58 +445,93 @@ class HtmlWebpackInlineSVGPlugin {
 
     }
 
-
     /**
      * append the inlineImages SVG data to the output HTML and remove the original img
-     * @param {string} html
-     * @param {Object} inlineImage - parse5 document
+     * @param {Object{}} html
+     * @param {Object{}} inlineImage - parse5 document
+     * @param {Object{}} data - SVG data
+     * @param {Object{}} resolve - Consumer's Promise resolve hook
      * @returns {Promise}
      *
      */
-    processOutputHtml (html, inlineImage) {
+    optimizeTheSvgRetrievedData ({ html, inlineImage, data, resolve }) {
```

Comment:
Perhaps a better name would just be `optimiseSvg`?
```suggestion
    optimiseSvg ({ html, inlineImage, data, resolve }) {
```